compress-diskimg.sh

```diff
@@ -50,5 +50,5 @@ fi
 echo -e "\e[1m[INFO]\e[0m Compressed disk image successfully, written to \"target/x86_64-arch/debug/bootimage-popcorn.bin.zst\"";
 
 # i mean those who work on popcorn must be nocturnal people so...
-echo -e "\e[1m[INFO]\e[0m Popcorn project wishes you a nice day!";
+echo -e "\e[1m[INFO]\e[0m PURE/popcorn project wishes you a nice day!";
```

Comment:
only popcorn as this is the kernel compressor not related to user space

Reply:
i meant the entire project's brand is that